main.cpp

```diff
@@ -4,7 +4,20 @@ using namespace std;
 
 int main()
 {
-  cout << "Hey Mel!";
+  int size;
+  char character;
+  cout << "Enter a size for your triangle";
+  cin  >>  size;
+  cout << "Enter a character: ";
+  cin  >>  character;
+  for(int i = 0; i < size; i++)
```

Comment:
I suggest that this for loop procedure should live in a separate function. A void function that takes in a char and size as parameters as follows `void generateEquilateralTriangle(char character, int size)`. In this way we can create separate triangle meme functions for different triangles.